moped-database/metadata/tables.yaml

```diff
@@ -4517,6 +4517,34 @@
 - table:
     name: moped_user_events
     schema: public
+  insert_permissions:
+    - role: moped-admin
+      permission:
+        check: {}
+        set:
+          user_id: x-hasura-user-db-id
```

Comment:
this is a column preset. something a little confusing about this is that using a preset revokes user permissions on the column. per [the docs](https://hasura.io/docs/latest/auth/authorization/permissions/column-presets/):

> the respective fields will also be removed from the generated GraphQL schema for that role.

which is exactly what we want. users cannot modify the `user_id` column on this table.